.github/workflows/protos-up-to-date.yaml

```diff
@@ -0,0 +1,28 @@
+name: Protobufs up to date
+run-name: make protos-up-to-date
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  merge_group:
```

Comment:
```suggestion
```

Unless these have to be here, for some reason?

Reply:
Yeah, we want the workflow to run on PRs. This needs to stay.

Reply:
Add a comment? This looks like nothing to the uninitiated.

Reply:
Comment added!